vision/beta/cloud-client/src/main/java/com/example/vision/AsyncBatchAnnotateImagesGcs.java

```diff
@@ -18,23 +18,16 @@
 
 // [START vision_async_batch_annotate_images_beta]
 import com.google.api.core.ApiFuture;
+import com.google.api.gax.longrunning.OperationFuture;
 import com.google.api.gax.paging.Page;
 import com.google.cloud.storage.Blob;
 import com.google.cloud.storage.Bucket;
 import com.google.cloud.storage.Storage;
 import com.google.cloud.storage.Storage.BlobListOption;
 import com.google.cloud.storage.StorageOptions;
-import com.google.cloud.vision.v1p4beta1.AnnotateImageRequest;
-import com.google.cloud.vision.v1p4beta1.AsyncBatchAnnotateImagesRequest;
+import com.google.cloud.vision.v1p4beta1.*;
```

Comment:
We don't like .* .   https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

Reply:
IDE decided to be helpful and collapse a long explicit list into a wildcard.